examples/api/test/material/selectable_region/selectable_region.0_test.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_api_samples/material/selectable_region/selectable_region.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  Future<void> sendKeyCombination(
+    WidgetTester tester,
+    LogicalKeyboardKey key,
+  ) async {
+    final LogicalKeyboardKey modifier = switch (defaultTargetPlatform) {
+      TargetPlatform.iOS || TargetPlatform.macOS => LogicalKeyboardKey.meta,
+      _ => LogicalKeyboardKey.control,
+    };
+    await tester.sendKeyDownEvent(modifier);
+    await tester.sendKeyDownEvent(key);
+    await tester.sendKeyUpEvent(key);
+    await tester.pump();
```

Comment:
I tried to put the pump at the end and it works.
Not sure why `sendKeyCombination` from packages/flutter/test/widgets/keyboard_utils.dart does not pump at the end.
No strong opinion on this, feel free to keep it as-is or move the pump.